src/components/task-plan/builder.cjsx

```diff
@@ -186,14 +218,20 @@ module.exports = React.createClass
         </BS.Col>
       </BS.Row>
 
-    if plan.type is 'homework'
-      feedbackNote = @renderFeedbackNote()
+    feedbackNote = '  Feedback will be released after the due date.' if plan.type is 'homework'
+    cannotEditNote = '  Open times cannot be editted after assignment is visible to students.' if @state.isVisibleToStudents
+
```

Comment:
Why do these have leading spaces?  If it's to ensure margin, shouldn't that be set via css?